### PR TITLE
floating discord icon

### PR DESCRIPTION
**Pull request by mruwnik**
image

### CODE REVIEW

File: app/root.css [hunk 1]
```diff
@@ -558,6 +558,29 @@ a:focus-visible,
   }
 }
 
+/* Begin Discord chat embed */
```
Comment (mruwnik) [inline app/root.css:561]: this is copied over from https://stampy.ai/wiki/MediaWiki:Common.css

File: app/root.css [hunk 2]
```diff
@@ -558,6 +558,29 @@ a:focus-visible,
...
+#discordChatBtn > svg{
+    -webkit-filter: drop-shadow( 1px 1px 1px rgba(0, 0, 0, .5));
+    filter: drop-shadow( 1px 1px 1px rgba(0, 0, 0, .5));
+    cursor: pointer;
+    position: fixed;
+    bottom: 1vw;
+    right: 2vw;
+    height: 3vw;
+    min-height: 40px;
+    min-width: 45px;
```
Comment (mruwnik) [inline app/root.css:572]: though this was added

File: app/routes/index.tsx [hunk 3]
```diff
@@ -126,6 +126,9 @@ export default function App() {
           />
         ))}
       </main>
+      <a id="discordChatBtn" href="https://discord.com/invite/7wjJbFJnSN">
```
Comment (Aprillion) [inline app/routes/index.tsx:129]: is it on purpose to use a different link that the one in footer few lines below, @plexish ?

**Review by Aprillion — APPROVED** [review]

File: app/routes/index.tsx (hunk 3 above)
Comment (mruwnik) [reply app/routes/index.tsx:129]: this one is an invite, while the other one just opens it, from what I understand

Comment (Aprillion) [reply app/routes/index.tsx:129]: the `https://discord.gg/vjFSCDyMCy` is also an invite, it redirects to `https://discord.com/invite/vjFSCDyMCy` (e.g. using https://unshorten.me)

Comment (plexish) [reply app/routes/index.tsx:129]: hm, yes, good point, they do go to different channels. I think ideally these people should be sent to #editing, which is actually https://discord.com/invite/Bt8PaRTDQC

Comment (mruwnik) [reply app/routes/index.tsx]: Updated URL

**Review by Aprillion — APPROVED** [review]

**Pull request merged**